### ISSUE TITLE
Bridge.net does not work on web worker

### ISSUE THREAD

**Issue by lidermanrony** [bridgedotnet/Bridge#3601]
# Version
17.0.0
# Description
Loading bridge.js  from a web worker throws the following exception:

```
VM29:1 Uncaught ReferenceError: window is not defined
    at eval (eval at <anonymous> (bridge.js:2016), <anonymous>:1:1)
    at bridge.js:2016
    at bridge.js:2026
    at bridge.js:41320
    at kustoWorker.js:46
    at Function.t._safeInvokeFunction (workerMain.js:24)
    at Function.t._invokeFactory (workerMain.js:24)
    at t.complete (workerMain.js:24)
    at s._onModuleComplete (workerMain.js:34)
    at s._resolve (workerMain.js:34)

```
Blame seems to lie on this specific line 
[this specific line](https://github.com/bridgedotnet/Bridge/blob/db9eca33c0c5ef97321be50744cb766eef58469c/Bridge/Resources/Core.js#L1998) that tries to access window (which isn't valid in web worker)

this used to work on an earlier version.

**Comment by geoffreymcgill** [comment]
Hi @lidermanrony. Looks like this was fixed in #3561 and will be available in the upcoming [Bridge 17.1.0](https://github.com/bridgedotnet/Bridge/milestone/60) release.

https://github.com/bridgedotnet/Bridge/pull/3562/files#diff-f1ef03b5078e642f9466d74dd667398dR2000

**Comment by lidermanrony** [comment]
Thanks @geoffreymcgill.

Is there some intermediate Nuget package i can install that contains this fix?
How far away is 17.0.1?
